### ISSUE TITLE
Sign binaries

### ISSUE THREAD

**Issue by ivankatliarchuk** [helm-unittest/helm-unittest#362]
Any interest in signing a release? Example for helm

<img width="1197" alt="Screenshot 2024-08-11 at 11 25 58" src="https://github.com/user-attachments/assets/51c1fff2-0b25-441c-83f1-459ae7dbb523">

This should helps users to verify the integrity and origin of a package.

**Comment by jkroepke** [comment]
As author of https://github.com/jkroepke/helm-secrets I can tell that this is absolutely not worth, since GPG is under-used.

**Comment by ivankatliarchuk** [comment]
This is quiet an old issue. I'm not getting what you mean with GPG is under used, aka operational overhead or something else. 

Anyway, hence is widely available now - Keyless signing backed by OIDC (GitHub Actions identity, etc.). So Sigstore/cosign is the nice to have choice - no key distribution problem, works seamlessly with GitHub Actions OIDC, and is now the CNCF/OpenSSF recommended approach.

**Comment by ivankatliarchuk** [comment]
Anyway, closing. Looks like it was implemented https://github.com/helm-unittest/helm-unittest/pull/859/changes

**Comment by jkroepke** [comment]
> Keyless signing backed by OIDC (GitHub Actions identity, etc.). So Sigstore/cosign is the nice to have choice ...

Yeah, but not supported by [Helm Provenance and Integrity](https://helm.sh/docs/topics/provenance/).
